src/System.Diagnostics.DiagnosticSource/dir.props

```diff
@@ -4,5 +4,7 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.2.0</AssemblyVersion>
     <IsNETCoreApp>true</IsNETCoreApp>
+    <IsUAP>true</IsUAP>
+    <IsUAPRef>false</IsUAPRef>
```

Comment:
sounds good, for now I only selected the ones required to satisfy the following:
 - Support our shipped UAP package.
 - All dependencies of netstandard.dll facade.
 - Required assemblies to have a full closure (Excluding classic assembly facades)